fabtests/test_configs/ofi_rxm/stress.json

```diff
@@ -23,33 +23,33 @@ This is a sample control file for fi_rdm_stress
 * offset is the starting location for data.
 * ms is time in milliseconds.
 
-Valid configration is in the JSON array defintion below.
+Valid configuration is in the JSON array definition below.
 "
 [
-
-{ "op" : "goodbye" },
-{ "op" : "hello" },
-{ "op" : "msg_req",    "size" : 1000 },
-{ "op" : "msg_resp" },
-{ "op" : "tag_req", "size" : 2000 },
-{ "op" : "tag_resp" },
-{ "op" : "msg_req",    "size" : 1000000 },
-{ "op" : "msg_resp" },
-{ "op" : "write_req",  "size" : 5600000, "offset" : 12000 },
-{ "op" : "write_resp" },
-{ "op" : "read_req",   "size" : 64000 },
-{ "op" : "read_resp" },
-{ "op" : "tag_req", "size" : 2000000},
-{ "op" : "tag_resp" },
-{ "op" : "read_req",   "size" : 32000 },
-{ "op" : "read_resp" },
-{ "op" : "write_req",  "size" : 86000, "offset" : 6000 },
-{ "op" : "write_resp" },
-{ "op" : "read_req",   "size" : 1000000 },
-{ "op" : "read_resp" },
-{ "op" : "write_req",  "size" : 56000, "offset" : 12000 },
-{ "op" : "write_resp" },
-{ "op" : "sleep",   "ms" : 100 },
-{ "op" : "exit" }
-
+	{ "op" : "goodbye" },
+	{ "op" : "hello" },
+	{ "op" : "msg_req", "size" : 1000 },
+	{ "op" : "msg_resp" },
+	{ "op" : "tag_req", "size" : 2000 },
+	{ "op" : "tag_resp" },
+	{ "op" : "msg_req", "size" : 1000000 },
+	{ "op" : "msg_resp" },
+	{ "op" : "write_req", "size" : 5600000, "offset" : 12000 },
+	{ "op" : "write_resp" },
+	{ "op" : "read_req", "size" : 64000 },
+	{ "op" : "read_resp" },
+	{ "op" : "tag_req", "size" : 2000000},
+	{ "op" : "tag_resp" },
+	{ "op" : "read_req", "size" : 32000 },
+	{ "op" : "read_resp" },
+	{ "op" : "write_req", "size" : 86000, "offset" : 6000 },
+	{ "op" : "write_resp" },
+	{ "op" : "read_req", "size" : 1000000 },
+	{ "op" : "read_resp" },
+	{ "op" : "write_req", "size" : 56000, "offset" : 12000 },
+	{ "op" : "write_resp" },
+	{ "op" : "sleep", "ms" : 100 },
+	{ "op" : "write_req", "size" : 10000000 },
+	{ "op" : "sleep", "ms" : 1 },
```

Comment:
Since you created the other two files, it is best to add back write_resp here so server side won't time out.
if you want to keep it as is, add a comment here so the user can know what to expect.  I tested with this and it works (even though it is no valid json):

        { "op" : "sleep", "ms" : 100 },
# Next write_req will casue a timeout on the server.  Server will continue after timeout.
        { "op" : "write_req", "size" : 10000000 },

Reply:
stupid markdown.  the comment is prefixed with "#" character.